cocos/3d/assets/morph-rendering.ts

```diff
@@ -473,7 +476,7 @@ class MorphUniforms {
  * @param vec4Capacity Capacity of vec4.
  */
 function createVec4TextureFactory (gfxDevice: Device, vec4Capacity: number) {
-    const hasFeatureFloatTexture = gfxDevice.hasFeature(Feature.TEXTURE_FLOAT);
+    const hasFeatureFloatTexture = gfxDevice.getFormatFeature(Format.RGBA32F) & (FormatFeatureBit.RENDER_TARGET | FormatFeatureBit.SAMPLED_TEXTURE);
```

Comment:
Morph rendering doesn't require the RT flag to be set, only sampled texture feature would be suffice